brainrender_napari/data_models/atlas_table_model.py

```diff
@@ -4,6 +4,7 @@
     get_local_atlas_version,
 )
 from napari.settings import get_settings
+from napari.settings._fields import Theme
```

Comment:
We should avoid importing from a private module. I'd rather make an exception and not provide a typehint if we don't have another option :thinking: